README.md

```diff
@@ -1 +1,169 @@
+> This project uses the following stack :
+
+<details>
+  <summary>Ruby</summary>
+  <summary>Ruby On Rails</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name='key-features'></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- [x] Creating a data model
+
+<p align='right'>(<a href='#readme-top'>back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name='getting-started'></a>
+
+<!-- > Clone the repository by clicking on the 'Code' button and copy the link -->
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+You should have Ruby install on your local machine
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+ https://github.com/Atril33/blog_app.git
+```
+
+### Install
+
+To install ruby go to : [Install Ruby](https://rubyinstaller.org/) and select your operating system.
+
+
+### Usage
+
+To run the project, open your project in your terminal
+
+```sh
+ bundle install
+```
+
+```sh
+ rails db:create
+```
+
+```sh
+ cd bin
+```
+
+```sh
+ rails db:migrate RAILS_ENV=development
+```
+
+```sh
+ rails server
+```
+
+
+<p align='right'>(<a href='#readme-top'>back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name='authors'></a>
+👤 **Syed Atril**
+
+- GitHub: [@githubhandle](https://github.com/Atril33)
+- Twitter: [@twitterhandle](https://twitter.com/AtrilSyed)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/syed-atril-831696248/)
+- Facebook [Facebook](https://web.facebook.com/profile.php?id=100073724910623)
+
+<p align='right'>(<a href='#readme-top'>back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name='future-features'></a>
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+- [ ] Validations and Model specs
+- [ ] Processing data in models
+- [ ] Setup and controllers
+- [ ] Controllers specs
+- [ ] Views
+- [ ] Forms
+- [ ] Integration specs for Views and fixing n+1 problems
+- [ ] Add Devise
+- [ ] Add authorization rules
+- [ ] Add API endpoints
+
+
+<p align='right'>(<a href='#readme-top'>back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name='contributing'></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/swarzstein/morse-code/issues).
+
+<p align='right'>(<a href='#readme-top'>back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name='support'></a>
+
+Give a ⭐️ if you like this project!
+
+<p align='right'>(<a href='#readme-top'>back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name='acknowledgements'></a>
+
+> I would like to say thanks to the Microverse team.
+
+<p align='right'>(<a href='#readme-top'>back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name='license'></a>
+
+This project is [MIT](https://github.com/Atril33/blog_app/blob/dev/LICENSE) licensed.
```

Comment:
- [ ] This link is broken. kindly consider fixing it as having broken links in our readme is unprofessional.

Reply:
Hi @Zuhaib042,
This link is working fine on my end I don't know how it doesn't work on your end. Isn't weird?
Please, check again I don't want to give anyone a bad feedback.